### ISSUE TITLE
Wrong behavior for descriptions

### ISSUE THREAD

**Issue by Cedrok** [ditrit/OGrEE-CLI#60]
For an OGrEE object, "description" is an array of strings.
The intended behavior is described [here](https://github.com/ditrit/OGrEE-3D/wiki/CLI-langage#modify-objects-attribute). To sumurize:
- [name].description=[value] => modify object.description[0]
- [name].descriptionX=[value] => modify object.description[X]   
If X is greater than the length of the array, it add 1 to the array's length and use that new index. (For example, we have only 2 descriptions and the user tries to add "description7=value": the CLI should set the value at description[2])

**Comment by Cedrok** [comment]
Test with API/checkVerificationLayer (commit 8f0c6e2) & CLI/descriptionFeature (commit f4b018175dc6bcb768b516d98dc82d1e54a01e4e)

Crash with `[name].description=[value]`
```
cedrok@OGrEE3D:/Physical$> test:description=coucou
OK. Attempting to update...
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
cli/controllers.UpdateObj({0xc0002874d0, 0xe}, {0x0, 0x0}, {0x0, 0x0}, 0xc0002906f0, 0x0)
        /OGrEE/OGrEE-CLI/controllers/commandController.go:419 +0x20f1
main.(*updateObjNode).execute(0xc00028c768)
        /OGrEE/OGrEE-CLI/ast.go:482 +0xb8
main.(*ast).execute(0xc00028c780)
        /OGrEE/OGrEE-CLI/ast.go:41 +0x69
main.InterpretLine(0xc000246000?)
        /OGrEE/OGrEE-CLI/repl.go:28 +0xc2
main.Repl(0xc00000e018, {0xc00015607d, 0x6})
        /OGrEE/OGrEE-CLI/repl.go:108 +0x1af
main.Start(0x6eb280?)
        /OGrEE/OGrEE-CLI/repl.go:92 +0x5c9
main.main()
        /OGrEE/OGrEE-CLI/main.go:109 +0x187e
```

**Comment by Cedrok** [comment]
Also, descriptions needs quotes for having spaces. Is it possible to eval everything after the "=" as a string ?

**Comment by Cedrok** [comment]
Crash fix in commit 6d523d24f061915879effbc4479a303f8700c016

**Comment by Cedrok** [comment]
Descriptions are broken, again. 
Here is what I did :
- Create an item (tenant here) & add a description with the "descriptionX" syntax: `item.description1=`
```
asher@OGrEE3D:/Physical$> +tn:test@123456
successfully created object
asher@OGrEE3D:/Physical$> test:description1=coucou
OK. Attempting to update...
Success
asher@OGrEE3D:/Physical$> get test
{
    "attributes": {
        "color": "123456"
    },
    "category": "tenant",
    "createdDate": "2022-12-20T15:44:45.89Z",
    "description": "[coucou]",
    "domain": "test",
    "id": "63a1d86de3e87718c87a8b33",
    "lastUpdated": "2022-12-20T15:44:54.78Z",
    "name": "test",
    "parentId": null
}
```
The description field is now a string with an array of something in it instead of an array of string

-  Change the value of the *description1*
```
asher@OGrEE3D:/Physical$> test:description1=value1
OK. Attempting to update...
Success
asher@OGrEE3D:/Physical$> get test
{
    "attributes": {
        "color": "123456",
        "description1": "value1"
    },
    "category": "tenant",
    "createdDate": "2022-12-20T15:44:45.89Z",
    "description": [
        "[coucou]",
        "value1"
    ],
    "domain": "test",
    "id": "63a1d86de3e87718c87a8b33",
    "lastUpdated": "2022-12-20T15:45:16.223Z",
    "name": "test",
    "parentId": null
}
```
The description field is now an array of strings, which is good. BUT doing description1 again is supposed to change the value of the first item of the description array, not adding one.  
A *description1* attribute also appeared. It shouldn't.

- Adding a new description with `item.description2=`
```
asher@OGrEE3D:/Physical$> test:description2=value2
OK. Attempting to update...
Success
asher@OGrEE3D:/Physical$> get test
{
    "attributes": {
        "color": "123456",
        "description1": "value1"
    },
    "category": "tenant",
    "createdDate": "2022-12-20T15:44:45.89Z",
    "description": "[[coucou],value1,value2]",
    "domain": "test",
    "id": "63a1d86de3e87718c87a8b33",
    "lastUpdated": "2022-12-20T15:45:43.732Z",
    "name": "test",
    "parentId": null
}
```
No description in attributes this time, this is good. 
*description* field is again a string with an array of something in it instead of an array of string.

- Modifying the *description1* with the simplier syntax: `item.decription=`
```
asher@OGrEE3D:/Physical$> test:description=value?
OK. Attempting to update...
Success
asher@OGrEE3D:/Physical$> get test
{
    "attributes": {
        "color": "123456",
        "description1": "value1"
    },
    "category": "tenant",
    "createdDate": "2022-12-20T15:44:45.89Z",
    "description": "value",
    "domain": "test",
    "id": "63a1d86de3e87718c87a8b33",
    "lastUpdated": "2022-12-20T15:46:51.178Z",
    "name": "test",
    "parentId": null
}
```
All descriptions are gone and the *description* field is a string (instead of an array of string)

**Comment by Cedrok** [comment]
This also make the CLI crash when description is wrongly formatted :
```
DEMO.ALPHA.B.R1.group2:label=#description
panic: interface conversion: interface {} is string, not []interface {}

goroutine 1 [running]:
cli/controllers.InteractObject({0xc0003704c0?, 0x522e3b?}, {0xc0004240f8, 0x5}, {0x4c9420, 0xc0006bc640}, 0x1)
        /OGrEE/OGrEE-CLI/controllers/commandController.go:2770 +0x785
main.(*updateObjNode).execute(0xc0002126c0)
        /OGrEE/OGrEE-CLI/ast.go:545 +0x772
main.(*ast).execute(0xc0006f4120)
        /OGrEE/OGrEE-CLI/ast.go:42 +0x62
main.InterpretLine(0x5bf5c0?)
        /OGrEE/OGrEE-CLI/repl.go:30 +0xc2
main.ExecuteFile(0xc0000c3930, {0xc0002a06c0, 0x34})
        /OGrEE/OGrEE-CLI/ocli.go:46 +0x1a5
main.LoadFile({0xc0002a06c0, 0x34})
        /OGrEE/OGrEE-CLI/ocli.go:89 +0x258
main.Repl(0xc00020c0c0, {0xc000010e40, 0x5})
        /OGrEE/OGrEE-CLI/repl.go:111 +0x185
main.Start(0xc00011dee8)
        /OGrEE/OGrEE-CLI/repl.go:102 +0x7cd
main.main()
        /OGrEE/OGrEE-CLI/main.go:92 +0xe70
```

**Comment by denizul** [comment]
This bug was fixed and merged into master as of 3c3517da3
